src/app/app.component.html

```diff
@@ -1,4 +1,4 @@
-<div class="main">
-    <app-searchbar></app-searchbar>
-    <app-container></app-container>
-</div>
+<div class="main" >
+    <app-searchbar (searchedPokemonInfo)="onSearch($event)"></app-searchbar>
+    <app-container [pokemonsInfo]="pokemonsInfo"></app-container>
```

Comment:
```suggestion
    <app-container [pokemonsInfo]="pokemonsInfo | async"></app-container>
```
penso non sia necessario in questo caso passare l'observable ma direttamente il risultato

inoltre, per convenzione gli observables ed i subjects chiamali con $ finale es. `pokemonsInfo$`